src/components/MultiGestureCanvas/index.tsx

```diff
@@ -124,6 +124,7 @@ function MultiGestureCanvas({
             pinchTranslateX.value = withSpring(0, SPRING_CONFIG);
             pinchTranslateY.value = withSpring(0, SPRING_CONFIG);
             zoomScale.value = withSpring(1, SPRING_CONFIG, callback);
+            pinchScale.value = 1;
```

Comment:
Why did you choose to use this approach instead of `pinchScale.value = withSpring(1, SPRING_CONFIG)`?